critiquebrainz/frontend/external/musicbrainz_db/testing.py

```diff
@@ -0,0 +1,12 @@
+import os
+from flask_testing import TestCase
+from critiquebrainz.frontend import create_app
+
+class MBDatabaseTestCase(TestCase):
```

Comment:
If creating a whole Flask app is unnecessary then I'd say it's better to just initialise the cache module.